src/containers/Users.js

```diff
@@ -0,0 +1,9 @@
+import Users from '../components/Users'
+import { connect } from 'react-redux'
+
+export default connect(
+  state => ({
+    userToAdd: state.addUser.userToAdd
+  }),
+  dispatch => ({})
+)(Users)
```

Comment:
Jezusmaria, musimy Was oduczyć tej nieczytelnej konstrukcji. Podejdź jutro, proszę, do mnie, to powiem Ci o co chodzi.